Formula/ocrmypdf.rb

```diff
@@ -130,8 +144,10 @@ def install
       venv.pip_install Pathname.pwd
     end
 
-    # Fix "ld: file not found: /usr/lib/system/libsystem_darwin.dylib" for lxml
-    ENV["SDKROOT"] = MacOS.sdk_path if MacOS.version == :sierra
+    on_macos do
+      # Fix "ld: file not found: /usr/lib/system/libsystem_darwin.dylib" for lxml
+      ENV["SDKROOT"] = MacOS.sdk_path if MacOS.version == :sierra
```

Comment:
Doesn't `MacOS.version == :sierra` always evaluate to false on Linux?

Reply:
That makes sense.  I'll test locally in Docker.

Reply:
As expected running this creates no problems on Linux, so I've removed the workaround there.